sigh/lib/assets/resign.sh

```diff
@@ -536,7 +536,7 @@ function resign {
         log "Updating the bundle version (CFBundleVersion) from '$CURRENT_VALUE' to '$BUNDLE_VERSION'"
         PlistBuddy -c "Set :CFBundleVersion $BUNDLE_VERSION" "$APP_PATH/Info.plist"
     fi
-
+    
```

Comment:
i had modified version to sign dylibs in main bundle pull request #6364 so i set it to the original version